src/main/java/net/sf/jabref/model/entry/BibEntry.java

```diff
@@ -57,6 +58,16 @@
      */
     private final Map<String, Set<String>> fieldsAsWords = new HashMap<>();
 
+    /*
```

Comment:
Good point. It's strange that Intellij does not distinguish between JavaDoc and Inline documentation in its color.